docs/dev/tools/common-errors.md

```diff
@@ -190,6 +190,18 @@ This can happen when there is a slow internet connection, when the module is try
   You can set these environment variables when you start `viam-server`, for instance `VIAM_MODULE_STARTUP_TIMEOUT=6m30 VIAM_RESOURCE_CONFIGURATION_TIMEOUT=3m0s viam-server -config example-machine.json`.
   Pass a sequence of numbers and time units, for example "6m30s50ms" for a timeout of 6 minutes, 30 seconds, and 50 milliseconds, or "5m" for a timeout of 5 minutes.
 
+## Common Micro-RDK errors
+
+### Unable to properly process stun response IceStunEncodingError
+
+**Full Error:** `unable to properly process stun response IceStunEncodingError`
+
+**Description:** This error occurs when something sent a stun message to a server and the response was eitehr not parseable or implied a different state than the client thought it was in.
```

Comment:
```suggestion
**Description:** Occurs when a client sends a STUN message to a server and receives one of the following problematic responses:
- the response cannot be parsed properly
- the response indicates a server state that conflicts with the client's understanding of the session state
```

[s] small typo fix (eitehr), reword for clarity
[opt] STUN is just niche enough that perhaps we should `{{< glossary_tooltip term_id="STUN" text="STUN" >}}` it and add a glossary entry?